packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -226,7 +226,11 @@ export default function DimensionsPanel( {
 	const contentSizeValue = decodeValue( inheritedValue?.layout?.contentSize );
 	const setContentSizeValue = ( newValue ) => {
 		onChange(
-			immutableSet( value, [ 'layout', 'contentSize' ], newValue )
+			immutableSet(
+				value,
+				[ 'layout', 'contentSize' ],
+				normalizeFalsyValue( newValue )
```

Comment:
I'm undecided between this and something explicit and immediate like `newValue || undefined`. While I understand the value of having a function like `normalizeFalsyValue` acting as an "anchor" (complete with documentation and the rationale for normalisation), I'm not sure that function name is descriptive enough.

That said, you could say I am picking nits; I'm wondering about how to make all the hidden traps of Global Styles as evident as possible.

Reply:
I agree that it's not a perfect solution, my main motivation is the documentation actually but I'm also undecided, especially since IMO we should be moving away slowly from its usage.

Reply:
Sorry I merged and missed this comment. I can remove that function soon if you feel strongly about it. I'm not sure how to handle the loss of documentation thought without a centralized function.

Reply:
Opened this https://github.com/WordPress/gutenberg/pull/50033

Reply:
Is this something we should document at a different place then - for example in the primary README of the global styles component in the block editor? I'm just not sure if documenting it within a separate helper function will help much with adopting it, especially when folks aren't aware that they should be using that function.